data/icon-overlay.js

```diff
@@ -240,12 +240,15 @@ function getTemplate() {
 
   containerEl.className = 'minvid__overlay__container';
   playIconEl.className = 'minvid__overlay__icon';
+  playIconEl.id = 'minvid__overlay__icon__play';
   playIconEl.title = 'Play Now';
-  addIconEl.className = 'minvid__overlay__icon__add';
+  addIconEl.className = 'minvid__overlay__icon';
+  addIconEl.id = 'minvid__overlay__icon__add';
```

Comment:
Neither of the issues linked reference overlay icon, so I'm not sure what these changes are addressing.